src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -694,7 +694,14 @@ message ObserveJobsQuery {
     // jobGroupDetail - job group details
     // jobGroupSequence - job group sequence
     // jobType - job type (batch or service)
-    // attributes - comma separated job attribute key/value pairs (for example "key1,key2:value2;k3:value3")
+    // attributes - comma separated job attribute key/value pairs. The same key may occur multiple times, with different
+    //              values (any value matches the filter). A value may be omitted, in which case if the key
+    //              occurs only once, only presence of the key is checked, without value comparison (otherwise the value
+    //              is an empty string). Example filters:
+    //              * 'key1' - matches, if the key is present
+    //              * 'key2:value2' - matches if the attributes contain key 'key2' with value 'value2'
+    //              * 'key3,key3:value3a,key3:value3b' - matches if the attributes contain key 'key3' with value '' or 'value3a' or 'value3b'
```

Comment:
I am confused about the first `key3` in this example. Because a user specified `key3` and no value, would it match on anything with the `key3` key or does the value have to be `''`?

Reply:
This is explained earlier: 
```
A value may be omitted, in which case if the key
occurs only once, only presence of the key is checked, without value comparison (otherwise the value
is an empty string)
```
So in this example `key3` is equivalent to `key3:<empty string>`

Reply:
This is explained earlier:
```
A value may be omitted, in which case if the key
occurs only once, only presence of the key is checked, without value comparison (otherwise the value
is an empty string)
So in this example key3 is equivalent to key3:<empty string>
```

Reply:
I read that but I still found it confusing. Basically it works differently when there is 1 versus many of the same key. I think it would have been more intuitive if you just explicitly checked for empty string and left the behavior the same in both cases.

Reply:
This is the current implementation. We should not change that,

Reply:
okay